<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Truva Linux Servis Yöneticisi</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QPushButton" name="pushButton" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>310</y>
      <width>83</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Başlat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2" >
    <property name="geometry" >
     <rect>
      <x>200</x>
      <y>310</y>
      <width>83</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Durdur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3" >
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>310</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Durum Bilgisi</string>
    </property>
   </widget>
   <widget class="QListView" name="listView" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
